### PR TITLE
Install "mini-redis-server" before running

### PR DESCRIPTION
**Pull request by Rezney**
Add instruction to install "mini-redis-server" before running it.

### CODE REVIEW

File: content/tokio/tutorial/hello-tokio.md [hunk 1]
```diff
@@ -54,6 +54,10 @@ pub async fn main() -> Result<()> {
 
 Make sure the Mini-Redis server is running. In a separate terminal window, run:
 
+```bash
+$ cargo install mini-redis-server
+```
+
 ```bash
 $ mini-redis-server
 ```
```
Comment (Darksonn) [inline content/tokio/tutorial/hello-tokio.md:63]: Maybe do it something like this?

>Make sure the Mini-Redis server is running. In a separate terminal window, run:
>```bash
>$ mini-redis-server
>```
>If you have not already installed mini-redis, you can do so with
>```bash
>$ cargo install mini-redis-server
>```

**Review by Darksonn — COMMENTED** [review]
We included this line on the previous page, although I'm happy to include it here too.

**Comment by Rezney** [comment]
Ahh, my bad. It is actually "Tutorial", not tutorials. I went directly to the "Hello world" without consulting the introduction. Feel free to close this and sorry for the noise. Btw, I fixed the PR as you suggested if you deem it can help somebody in rush as me :)

**Review by Darksonn — APPROVED** [review]
People link to specific pages all the time. I think making them more self-contained can be useful, at least to some extent.

**Pull request merged**